linchpin/hooks/action_managers/ansible_action_manager.py

```diff
@@ -195,6 +195,8 @@ def execute(self):
             e_vars = action.get("extra_vars", {})
             extra_vars.update(e_vars)
             if self.context:
-                extra_vars["linchpin_context"] = self.get_ctx_params()
```

Comment:
@herlo @seandst 
This should fix the inventory_file problem mentioned by @seandst . 
Now the inventory_file is passed as a extra_var parameter such that , it will be available for the playbooks .